openapi/openapi.yaml

```
@@ -2,7 +2,7 @@ openapi: "3.0.0"
 info:
   title: "ExplainaBoard"
   description: "Backend APIs for ExplainaBoard"
-  version: "0.2.8"
+  version: "0.2.9"
```

Maybe this needs to be bumped to 0.2.10, as we've already used 0.2.9.
```suggestion
  version: "0.2.10"
```